models/articles-models.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection");
+
+exports.fetchArticleById = (article_id) => {
+  if (isNaN(article_id)) {
```

Comment:
PSQL will send back an error for you if the article_id is invalid. Don't do your own Promise.reject for this, instead, create a psql error handler and differentiate them based on the error codes (maybe check can-you-handle-this sprint or Mitch's rare treasures to guide you)